storage/api/Storage/Storage.cs

```diff
@@ -705,6 +707,43 @@ private void GenerateSignedUrl(string bucketName, string objectName)
         }
         // [END storage_generate_signed_url]
 
+        // [START storage_generate_signed_url_v4]
+        private void GenerateV4SignedGetUrl(string bucketName, string objectName)
+        {
+            UrlSigner urlSigner = UrlSigner
+            .FromServiceAccountPath(Environment.GetEnvironmentVariable("GOOGLE_APPLICATION_CREDENTIALS"))

```

Comment:
I'd indent these lines by an extra four spaces if possible. If that makes the line too long, you could fetch the credentials path into a separate variable first.